src/components/Expenses/ExpenseFilter.tsx

```diff
@@ -1,8 +1,14 @@
 import React from "react";
 import "./ExpenseFilter.css";
 
-const ExpenseFilter = ({ showFilteredByYear, selectedYear }) => {
-  const onFilterChange = ({ target }) => {
+const ExpenseFilter = ({
+  showFilteredByYear,
+  selectedYear,
+}: {
+  showFilteredByYear: (arg: string) => void;
+  selectedYear: string;
+}) => {
+  const onFilterChange = ({ target }: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
There is no need to import the whole React class. You can import only the needed types for it. In this case, you can just `ChangeEvent<HTMLSelectElement>` and it will import correctly.